src/sage/groups/perm_gps/partn_ref/data_structures.pxd

```diff
@@ -136,6 +136,35 @@ cdef inline void OP_join(OrbitPartition *OP, int m, int n) noexcept:
     if m_root != n_root:
         OP.num_cells -= 1
 
+
+cdef inline void OP_make_set(OrbitPartition *OP) noexcept:
+    cdef int i, n = OP.degree
+    cdef int *new_parent, *new_rank, *new_mcr, *new_size
+
+    cdef int *int_array = <int *> sig_malloc( 4*(n+1) * sizeof(int) )
+    if int_array is not NULL:
+        OP.degree = n + 1
+        OP.num_cells = OP.num_cells + 1
+        new_parent = int_array
+        new_rank = int_array + (n + 1)
+        new_mcr = int_array + (2*n + 2)
+        new_size = int_array + (3 * n + 3)
+
+        memcpy(new_parent, OP.parent, n * sizeof(int) )
+        memcpy(new_rank, OP.rank, n * sizeof(int) )
+        memcpy(new_mcr, OP.mcr, n * sizeof(int) )
+        memcpy(new_size, OP.size, n * sizeof(int) )
+
+        new_parent[n] = n
+        new_rank[n] = 0
+        new_mcr[n] = n
+        new_size[n] = 1
+
+        OP.parent = new_parent
+        OP.rank   = new_rank
```

Comment:
please avoid the extra spaces before `=`. This is not compatible with Python coding style.